### PR TITLE
[UXE-1888] feat: add clicked to create origin event 

### PR DESCRIPTION
**Pull request by pauloSF0**
## Pull Request
[UXE-1888] 
### What is the new behavior introduced by this PR?

add clicked to create origin event 
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
Captura de Tela 2024-03-05 às 14 14 38

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

[UXE-1888]: https://aziontech.atlassian.net/browse/UXE-1888?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/plugins/adapters/AnalyticsTrackerAdapter.js [hunk 1]
```diff
@@ -292,6 +292,17 @@ export class AnalyticsTrackerAdapter {
     return this
   }
 
+  /**
+   * @returns {AnalyticsTrackerAdapter}
+   */
+  clickedToCreateOrigin() {
+    this.#events.push({
+      eventName: 'Clicked to Create Origin',
```
Comment (peterpaulo-azion) [inline src/plugins/adapters/AnalyticsTrackerAdapter.js:300]: Já existe um evento chamado "ClickedToCreate" que recebe o nome do produto ao lado, foi utilizado em Edge Application.

Ele recebe ${payload.productName}. Utilizar ele nesse caso ao invés de criar um novo.

File: src/tests/plugins/adapters/AnalyticsTrackerAdapter.test.js [hunk 2]
```diff
@@ -327,4 +327,14 @@ describe('AnalyticsTrackerAdapter', () => {
 
     expect(analyticsClientSpy.track).toHaveBeenCalledWith('Failed to Deploy', {})
   })
+
+  it('should use the clickedToCreateOrigin event', () => {
+    const { sut, analyticsClientSpy } = makeSut()
+
+    sut.clickedToCreateOrigin({})
```
Comment (peterpaulo-azion) [inline src/tests/plugins/adapters/AnalyticsTrackerAdapter.test.js:334]: Ajustar testes conforme o outro comentário.

File: src/views/EdgeApplicationsOrigins/Drawer/index.vue [hunk 3]
```diff
@@ -160,6 +162,10 @@
     showEditOriginDrawer.value = false
   }
 
+  const handleTrackCreation = () => {
+    tracker.clickedToCreateOrigin().track()
```
Comment (peterpaulo-azion) [inline src/views/EdgeApplicationsOrigins/Drawer/index.vue:166]: Utilizar outro evento clickedToCreate(), conforme outro comentário.

**Review by peterpaulo-azion — CHANGES_REQUESTED** [review]

File: src/tests/plugins/adapters/AnalyticsTrackerAdapter.test.js (hunk 2 above)
Comment (pauloSF0) [reply src/tests/plugins/adapters/AnalyticsTrackerAdapter.test.js:334]: verdade, vlw ai

**Review by peterpaulo-azion — APPROVED** [review]

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**